src/transformers/models/seaformer/convert_seaformer_original_to_pytorch.py

```diff
@@ -0,0 +1,208 @@
+# coding=utf-8
+# Copyright 2023 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Convert Seaformer checkpoints."""
+
+
+import argparse
+import json
+from collections import OrderedDict
+from pathlib import Path
+
+import requests
+import torch
+from huggingface_hub import hf_hub_download
+from PIL import Image
+
+from transformers import (
+    SeaformerConfig,
+    SeaformerFeatureExtractor,
```

Comment:
We are replacing all `XXXFeatureExtractor` classes with `XXXImageProcessor`
```suggestion
    SeaformerImageProcessor,
```